allergy intolerances/README.md

```diff
@@ -1 +1,168 @@
+                        }
+                    ],
+                    "system": "http://snomed.info/sct",
+                    "code": "395102008",
+                    "display": "H/O: aspirin allergy"
+                }
+            ],
+            "text": "H/O: aspirin allergy"
+        },
+        "patient": {
+            "reference": "Patient/180b44bf-31d8-407b-b8ca-994a3f4a226c"
+        },
+        "onsetDateTime": "2010-01-13",
+        "assertedDate": "2010-01-13",
+        "recorder": {
+            "reference": "Practitioner/3707E1F0-9011-11EC-B1E5-0800200C9A66"
+        },
+        "asserter": {
+            "reference": "Practitioner/3707E1F0-9011-11EC-B1E5-0800200C9A66"
+        },
+        "note": [
+            {
+                "text": "Drug Allergy - Apsrin"
+            }
+        ]
+    }
+},
+```
+</details>
+
+## JSON FHIR > XML HL7
+
+An Allergy Intolerance is mapped to a CompoundStatement with inner ObservationStatement.
+
+| Mapped to (XML HL7)                                                          | Mapped from (JSON FHIR / other source )                        |
+|------------------------------------------------------------------------------|----------------------------------------------------------------|
+| CompoundStatement / id \[@root]                                              | `AllergyIntolerance.id`                                        |
+| CompoundStatement / code                                                     | `AllergyIntolerance.category[0]`                               |
+| CompoundStatement / effectiveTime                                            | `AllergyIntolerance.onsetDateTime`                             |
+| CompoundStatement / availabilityTime                                         | `AllergyIntolerance.assertedDate`                              |
+| ObservationStatement / id \[@root]                                           | `AllergyIntolerance.id`                                        |
+| ObservationStatement / code                                                  | `AllergyIntolerance.code`                                      |
+| ObservationStatement / effectiveTime                                         | `AllergyIntolerance.onsetDateTime`                             |
+| ObservationStatement / availabilityTime                                      | `AllergyIntolerance.assertedDate`                              |
+| ObservationStatement / pertinentInformation / pertinentAnnotation / text     | `AllergyIntolerance.note\[0].text`                             |
+| ObservationStatement / Participant \[@typeCode=AUT] / agentRef / id \[@root] | `AllergyIntolerance.recorder`                                  |
+| ObservationStatement / Participant \[@typeCode=PRF] / agentRef / id \[@root] | `AllergyIntolerance.asserter` or `AllergyIntolerance.recorder` |
+
+<details><summary>Example XML</summary>
+
+```
+<component typeCode="COMP" contextConductionInd="true">
```

Comment:
There is no compound statement in this example, but is mentioned in the mapping

Reply:
This should be a generated XML from the GP2GP, currently a sample input for Patient Switching adaptor.